### ISSUE TITLE
[🐛 Bug ]: Windows 11 -> Vmware -> Ubuntu 22 -> Docker -> BOOTING

### ISSUE THREAD

**Issue by ErcinDedeoglu** [budtmo/docker-android#405]
### Operating System

Ubuntu 22.04

### Docker Image

budtmo/docker-android:emulator_11.0

### Expected behaviour

1. My host machine is Windows 11 and VMware Workstation 17.5.
2. Ubuntu 22.04 LTS installed on VMware.
3. Docker installed on Ubuntu 22.04.

**-> docker ps**
```
CONTAINER ID   IMAGE                                 COMMAND                  CREATED         STATUS         PORTS                                                                                    NAMES
9e30fb34a595   budtmo/docker-android:emulator_11.0   "/home/androidusr/do…"   6 minutes ago   Up 6 minutes   4723/tcp, 5554-5555/tcp, 5900/tcp, 9000/tcp, 0.0.0.0:6080->6080/tcp, :::6080->6080/tcp   android-container
```

**-> device_status**
```
root@ubuntu:/home/sa# docker exec -it android-container cat device_status
BOOTINGroot@ubuntu:/home/sa# 
```

**-> kvm**
```
root@ubuntu:/home/sa# kvm-ok
INFO: /dev/kvm exists
KVM acceleration can be used
root@ubuntu:/home/sa# 
```

When I open http://192.168.0.62:6080/
it successfully connecting to VNC but not showing the device as expected.

I executed 
docker run -d -p 6080:6080 -e EMULATOR_DEVICE="Samsung Galaxy S10" -e WEB_VNC=true --device /dev/kvm --name android-container budtmo/docker-android:emulator_11.0

### Actual behaviour

It's hanging on BOOTING state and http://192.168.0.62:6080/ address not showing the device.

### Logs

root@ubuntu:/home/sa# docker logs -f 9e --tail 1000
2024-01-27 07:47:58,265 INFO supervisord started with pid 8
2024-01-27 07:47:58,265 INFO supervisord started with pid 9
2024-01-27 07:47:58,266 INFO supervisord started with pid 7
2024-01-27 07:47:59,270 INFO spawned: 'android_port_forward' with pid 13
2024-01-27 07:47:59,270 INFO spawned: 'd_screen' with pid 15
2024-01-27 07:47:59,270 INFO spawned: 'appium' with pid 14
2024-01-27 07:47:59,273 INFO spawned: 'device' with pid 16
2024-01-27 07:47:59,273 INFO spawned: 'd_wm' with pid 17
2024-01-27 07:47:59,275 INFO spawned: 'log_web_shared' with pid 18
2024-01-27 07:47:59,285 INFO spawned: 'vnc_server' with pid 19
2024-01-27 07:47:59,286 INFO spawned: 'vnc_web' with pid 20
2024-01-27 07:47:59,434 INFO exited: log_web_shared (exit status 0; not expected)
2024-01-27 07:47:59,435 INFO exited: appium (exit status 0; not expected)
2024-01-27 07:48:00,417 INFO success: android_port_forward entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-01-27 07:48:00,431 INFO success: d_screen entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-01-27 07:48:00,431 INFO success: d_wm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-01-27 07:48:00,432 INFO success: vnc_server entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-01-27 07:48:00,432 INFO success: vnc_web entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-01-27 07:48:00,436 INFO gave up: appium entered FATAL state, too many start retries too quickly
2024-01-27 07:48:00,436 INFO success: device entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-01-27 07:48:00,436 INFO spawned: 'log_web_shared' with pid 65
2024-01-27 07:48:00,583 INFO exited: log_web_shared (exit status 0; not expected)
2024-01-27 07:48:03,574 INFO spawned: 'log_web_shared' with pid 158
2024-01-27 07:48:03,720 INFO exited: log_web_shared (exit status 0; not expected)
2024-01-27 07:48:06,727 INFO spawned: 'log_web_shared' with pid 160
2024-01-27 07:48:06,882 INFO exited: log_web_shared (exit status 0; not expected)
2024-01-27 07:48:07,883 INFO gave up: log_web_shared entered FATAL state, too many start retries too quickly

**Comment by Halphas** [comment]
same :(

**Comment by yasselbhdev** [comment]
same here with a dedicated Ubuntu server 22.04

**Comment by yasselbhdev** [comment]
root@server:/home/demon# docker logs -f b8f3f9c977f7 --tail 1000
2024-01-28 01:03:45,360 INFO supervisord started with pid 9
2024-01-28 01:03:45,363 INFO supervisord started with pid 8
2024-01-28 01:03:45,393 INFO supervisord started with pid 7
2024-01-28 01:03:46,363 INFO spawned: 'appium' with pid 13
2024-01-28 01:03:46,365 INFO spawned: 'device' with pid 14
2024-01-28 01:03:46,366 INFO spawned: 'android_port_forward' with pid 15
2024-01-28 01:03:46,367 INFO spawned: 'log_web_shared' with pid 16
2024-01-28 01:03:46,396 INFO spawned: 'd_screen' with pid 17
2024-01-28 01:03:46,399 INFO spawned: 'd_wm' with pid 18
2024-01-28 01:03:46,402 INFO spawned: 'vnc_server' with pid 19
2024-01-28 01:03:46,405 INFO spawned: 'vnc_web' with pid 20
2024-01-28 01:03:46,924 INFO exited: log_web_shared (exit status 0; not expected)
2024-01-28 01:03:46,928 INFO exited: appium (exit status 0; not expected)
2024-01-28 01:03:46,993 INFO exited: vnc_server (exit status 1; not expected)
2024-01-28 01:03:47,517 INFO gave up: appium entered FATAL state, too many start retries too quickly
2024-01-28 01:03:47,517 INFO success: device entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-01-28 01:03:47,883 INFO success: android_port_forward entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-01-28 01:03:47,989 INFO success: d_screen entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-01-28 01:03:47,989 INFO success: d_wm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-01-28 01:03:47,989 INFO success: vnc_web entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-01-28 01:03:48,363 INFO spawned: 'log_web_shared' with pid 130
2024-01-28 01:03:48,863 INFO exited: log_web_shared (exit status 0; not expected)
2024-01-28 01:03:48,996 INFO spawned: 'vnc_server' with pid 151
2024-01-28 01:03:50,406 INFO success: vnc_server entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-01-28 01:03:51,297 INFO spawned: 'log_web_shared' with pid 169
2024-01-28 01:03:51,664 INFO exited: log_web_shared (exit status 0; not expected)
2024-01-28 01:03:54,671 INFO spawned: 'log_web_shared' with pid 171
2024-01-28 01:03:54,976 INFO exited: log_web_shared (exit status 0; not expected)
2024-01-28 01:03:55,977 INFO gave up: log_web_shared entered FATAL state, too many start retries too quickly

**Comment by yasselbhdev** [comment]
ohhh I found a post here with the same issue, it seems disk space:
https://github.com/budtmo/docker-android/issues/365

**Comment by yasselbhdev** [comment]
> ohhh I found a post here with the same issue, it seems disk space: #365

this was the solution with image budtmo/docker-android:emulator_11.0

**Comment by Halphas** [comment]
yeah i even get on the other images the same error. cant start the emu. is somebody able to ? then i know at least that its my personal issue :D

**Comment by Halphas** [comment]
is this project dead ? :(

**Comment by fengluo2** [comment]
I also encountered this problem.

**Comment by gl953236368** [comment]
I also, but is ubuntu 24.04

**Comment by gl953236368** [comment]
> I also, but is ubuntu 24.04

enter the vnc and open the terminal，if everying is ok that the emulator will be worked，but will display u error;
`emulator -list-avds # check emulator lists
emulator -avd  [emulator] # emulator name  eg. samsung_galaxy_s6_9.0 `
